test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,237 @@
+const { strict: assert } = require('assert');
+const {
+  convertToHexValue,
+  withFixtures,
+  switchToNotificationWindow,
+  openDapp,
+  unlockWallet,
+  WINDOW_TITLES,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+const recipientAddress = '0x2f318C334780961FB129D2a6c30D0763d9a5C970';
+const editGasfeeCustomToken = async (driver) => {
+  const inputs = await driver.findElements('input[type="number"]');
+  const gasLimitInput = inputs[0];
+  const gasPriceInput = inputs[1];
+  await gasLimitInput.clear();
+  await gasLimitInput.fill('60000');
+  await gasPriceInput.clear();
+  await gasPriceInput.fill('10');
+  await driver.clickElement({ text: 'Save', tag: 'button' });
+};
+
+describe('Transfer custom tokens @no-mmi', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
```

Comment:
Can you try to use `defaultGanacheOptions` from `helpers` to reduce duplicated code? Thanks !

Reply:
Done, thanks!